kythe/java/com/google/devtools/kythe/platform/java/filemanager/CompilationUnitPathFileManager.java

```diff
@@ -200,6 +205,8 @@ public boolean handleFileManagerOption(Option unused, String value) {
   @Override
   public void close() throws IOException {
     super.close();
+    fileSystem.close();
```

Comment:
Hygienically these should all be in try/finallys, e.g.
```
boolean threw = true;
try { super.close(); threw = false; }
finally {
  try { Closeables.close(fileSystem, threw); threw = false; }
  finally { Closeables.close(memFileSystem, threw); }
}

but that's ugly and possibly overkill.

Reply:
That was my first thought, but after your comment I decided to see if Guava had any helpful utilities for accomplishing this et voila!